web/posts/2017-07-25-making-time-for-side-projects.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: Making Time for Side Projects
+tags: development, life
+---
+
+I have about a million things I'd like to do at any given moment. If I sit
+down and try to choose something to spend my time on, it's fairly
+overwhelming. There are blog posts I could write, books I could read, movies
+and TV shows I could watch, or any number of ways I could continue my own
+education via videos or articles. And then there are the side projects. So so
+many side projects.
+
+The problem with side projects is that you can't complete them. No matter what
+you think, side projects aren't done. You might be able to finish reading a
```

Comment:
Here's a post about that, inspired by our good mutual friend Edward Q. Loveall: http://gabebw.com/blog/2015/01/14/move-your-goalposts